securedrop_client/storage.py

```diff
@@ -937,7 +941,14 @@ def _delete_source_collection_from_db(session: Session, source: Source) -> None:
             logger.error("Could not add source {} to deletedconversation table".format(source.uuid))
             session.rollback()
 
-    session.commit()
+    try:
+        session.commit()
+    except SQLAlchemyError:
+        logger.error(
```

Comment:
Would be helpful to print the original error message as well here.

Reply:
Re error-logging: fixed in cda3150